docker/clone-and-preflight.ts

```diff
@@ -65,14 +65,28 @@ if (projectUsesPostgresql) {
   // Example script:
   // https://github.com/upleveled/preflight-test-project-next-js-passing/blob/e65717f6951b5336bb0bd83c15bbc99caa67ebe9/scripts/alpine-postgresql-setup-and-start.sh
   const postgresUid = Number((await execa`id -u postgres`).stdout);
-  await execa({
+  const postgresProcess = await execa({
     // postgres user, for initdb and pg_ctl
     uid: postgresUid,
     // Show output to simplify debugging
-    stdout: 'inherit',
+    stdout: 'pipe',
     stderr: 'inherit',
   })`bash ./scripts/alpine-postgresql-setup-and-start.sh`;
 
+  const preflightEnvironmentVariables = postgresProcess.stdout.match(
+    /PREFLIGHT_ENVIRONMENT_VARIABLES:\n(\[.*?\])/,
+  )?.[1];
+
+  if (preflightEnvironmentVariables) {
+    const environmentVariablesKeys: string[] = JSON.parse(
+      preflightEnvironmentVariables,
+    );
+
+    for (const key of environmentVariablesKeys) {
+      process.env[key] = `fake_${key.toLowerCase()}`;
```

Comment:
1. maybe 'fake_' isn't expressive enough for a student debugging why they're app isn't working (eg. if their app actually needs the real env var value)

   so probably best to make the prefix or value more expressive

2. you probably can get rid of the `key.toLowerCase()` suffix too

Reply:
Removed the `fake` prefix, and changed it to `MISSING_ENV_`, so when students debug, they would see `MISSING_ENV_CLOUDINARY_API_KEY` instead of `fake_cloudinary_api_key` 
[5a07f4e](https://github.com/upleveled/preflight/pull/632/commits/5a07f4eba05713140be6e6778bc00541d34f24ca)

Reply:
hmm, `MISSING_ENV_` is not much more helpful than `fake_` - I guess we should be more expressive and make sure students can easily find the source of this (probably should mention Preflight and maybe UpLeveled)

probably also with some kind of mention of "fake" or "dummy" or "placeholder"

Reply:
Changed now to `UPLEVELED_PREFLIGHT_PLACEHOLDER`, I think students will understand that this comes from Upleveled and is for Preflight as a placeholder.
[7dcffd4](https://github.com/upleveled/preflight/pull/632/commits/7dcffd44c81265c2fbc2743c721f4383173d7293)